tests/AnalyzerTests/GlobalConfig/SadTests.cs

```diff
@@ -178,7 +178,7 @@ public async Task Not_valid_conversion()
         var source = @"using System;
 using Vogen;
 
-[assembly: VogenDefaults(conversions: (Conversions)666)]
+[assembly: VogenDefaults(conversions: (Conversions)4242)]
```

Comment:
Previously, only 8 bits of the `Conversions` flag were used, so `666` was invalid. Not now there's 9 bits, we needed a higher invalid value (over `1024`). Took me a while to figure out why those tests were failing, but I was pleased that they were failing! :)